File: CS/TestValidation.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;
using DevExpress.Persistent.BaseImpl;


namespace TestValidation.Module {
    public sealed partial class TestValidationModule : ModuleBase {
        public TestValidationModule() {
            ModelDifferenceResourceName = "TestValidation.Module.Model.DesignedDiffs";
            InitializeComponent();
        }
        public override void Setup(XafApplication application) {
            XafTypesInfo.Instance.RegisterEntity("Employee", typeof(IEmployee));
            base.Setup(application);
        }
    }
}

File: VB/TestValidation.Module/Module.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic

Imports DevExpress.ExpressApp
Imports System.Reflection
Imports DevExpress.Persistent.BaseImpl


Namespace TestValidation.Module
	Public NotInheritable Partial Class TestValidationModule
		Inherits ModuleBase
		Public Sub New()
			ModelDifferenceResourceName = "TestValidation.Module.Model.DesignedDiffs"
			InitializeComponent()
		End Sub
		Public Overrides Sub Setup(ByVal application As XafApplication)
			XafTypesInfo.Instance.RegisterEntity("Employee", GetType(IEmployee))
			MyBase.Setup(application)
		End Sub
	End Class
End Namespace
